hummingbot/market/markets_recorder.py

```diff
@@ -15,6 +16,7 @@
     Union
 )
 
+from hummingbot.client.settings import CONF_FILE_PATH
```

Comment:
Would this save the file in the `conf` folder? If possible, we should save this in the `data` folder since that's where the sqlite db containing the trades are.

Reply:
Done!

Reply:
Done!